src/JustEat.StatsD/TimerExtensions.cs

```diff
@@ -5,7 +5,7 @@ namespace JustEat.StatsD
 {
     public static class TimerExtensions
     {
-        public static IDisposable StartTimer(this IStatsDPublisher publisher, string bucket)
+        public static IDisposableTimer StartTimer(this IStatsDPublisher publisher, string bucket = null)
```

Comment:
This feels like a stealthy way to cancel a stat, which could lead to suprising behaviour (lost into the aether) if the dev forgets to initialise a bucket.  It'd probably be better to either a) insist on a non-null-or-empty value for the bucket here (and in the setter), or b) raise an exception if the bucket is null at the end.  If anyone absolutely feels the need for a cancel() use case, we can add it explicitly later.